content/hi/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -21,66 +21,66 @@ sitemap:
 कुबेरनेट्स नाम ग्रीक से उत्पन्न हुआ है, जिसका अर्थ है हेल्समैन या पायलट।
 K8s एक संक्षिप्त नाम के रूप में "K" और "s" के बीच आठ अक्षरों को गिनने का परिणाम है। Google ने 2014 में कुबेरनेट्स प्रोजेक्ट को ओपन-सोर्स किया। कुबेरनेट्स [गूगल के 15 से अधिक वर्षों के अनुभव](/blog/2015/04/borg-predecessor-to-kubernetes/) को समुदाय से सर्वोत्तम नस्ल के विचारों और प्रथाओं के साथ बड़े पैमाने पर उत्पादन कार्यभार को जोड़ती है।
 
-## काल में वापस जाना
+## समय में वापस जाना
 
-आइए काल में वापस जाकर एक नज़र डालते हैं कि कुबेरनेट्स इतना उपयोगी क्यों है।
+आइए वक्त में पलट कर एक दृष्टि डालते हैं कि कुबेरनेट्स इतना उपयोगी क्यों है।
```

Comment:
```suggestion
आइए वक्त में पीछे जाकर एक नज़र डालते हैं कि कुबेरनेट्स इतना उपयोगी क्यों है।
```

Reply:
दृष्टि is better than  नज़र .I think " आइए वक्त में पीछे जाकर एक दृष्टि डालते हैं "